### PR TITLE
chore: do not collapse categories on sidebar

### LINKED ISSUE #5338: Doc: Improve left-sidebar navigation
### Is your enhancement related to a problem? Please describe

When you arrive on the documentation, we have a lot of sections but it's hard to know what's in each of the section. It forces the reader to "unflod" each of the section to have an idea of what's into it. 

Screenshot 2023-12-20 at 10 18 44

### Describe the solution you'd like

We would like to have 2 levels of sections unfolded by default.
Also, we would like to visually differentiate the levels of navigation.

Example:
Screenshot 2023-12-20 at 10 21 16
(link: https://ionicframework.com/docs)

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

chore: disable autoCollapseCategories on sidebar, as requested in #5338.

### What issues does this PR fix or reference?

fixes #5338

### CODE REVIEW

File: website/docusaurus.config.js
Comment (benoitf) [inline website/docusaurus.config.js]: maybe just drop the full line as it's the default behaviour

Comment (themr0c) [reply website/docusaurus.config.js]:
```suggestion
```

**Comment by themr0c** [comment]
@deboer-tim Does it correspond to your intentions in #5338?

**Review by benoitf — APPROVED** [review]

**Comment by deboer-tim** [comment]
> @deboer-tim Does it correspond to your intentions in #5338?

That's Stevan's issue, but it asks to initially expand only 2 levels deep (not all sections) and to change the styling of main sections vs individual docs. This PR doesn't do either of those yet, so I don't believe it should be listed as fixing the issue.

**Comment by themr0c** [comment]
Oh @deboer-tim your comment showed up after I hit the merge button. I apologize. 2 levels deep is not an achievable option, AFAIK.

**Pull request merged**
